core/src/main/java/com/yahoo/ycsb/DB.java

```diff
@@ -41,97 +41,94 @@
  * target application.  For the sake of comparison between experiments we also 
  * recommend you explain the semantics you chose when presenting performance results.
  */
-public abstract class DB
-{
-	/**
-	 * Properties for configuring this DB.
-	 */
-	Properties _p=new Properties();
+public abstract class DB {
+  /**
+   * Properties for configuring this DB.
+   */
+  private Properties properties = new Properties();
```

Comment:
So looking at this closer, there already exist get/set property methods so that should encapsulate the _p variable and allow us to change the name. If people were relying on the _p variable they should have been using the get/set methods.